### DISCUSSION TITLE
Smart Contract Development

### DISCUSSION THREAD

**Discussion by clever-web** [Web-development-s/Stripe-with-laravel#18] [Q&A] [answered]
Please let me know about solidity. How to learn it?

**Comment by prowebdev119** [comment]
Hello
I am happy to answer to you.
Solidity is a contract-oriented, high-level programming language for implementing smart contracts. 

Solidity is highly influenced by C++, Python and JavaScript and has been designed to target the Ethereum Virtual Machine (EVM).

Solidity is statically typed, supports inheritance, libraries and complex user-defined types programming language.

You can use Solidity to create contracts for uses such as voting, crowdfunding, blind auctions, and multi-signature wallets.

**Comment by superstar1205** [comment, marked as answer]
Yeah
